lint.sh

```diff
@@ -29,23 +29,45 @@ check_jjb(){
     || { echo "JJB Syntax fail"; rc=1; }
 }
 
+# This pulls job.dsl from jjb templates so they can be checked for groovy syntax
+extract_groovy_from_jjb(){
+  mkdir -p tmp_groovy
+  pushd rpc_jobs
+  for i in *.yml
+  do
+    python <<EOF
```

Comment:
I do see the irony of adding an embeded script in a PR for extracting embedded scripts :/ I'll move it out.